es-bs-base/scss/_nav.scss

```diff
@@ -179,9 +179,9 @@ $nav-hover-delay: 300ms;
     font-size: $font-size-75;
   }
 
-  // All links should have font-weight-normal (weight 400)
+  // All links and labels in the navigation should start at a font-weight-normal (weight 400)
   // TODO: This overrides styles for `eyebrow`
-  a {
+  a, label {
```

Comment:
Stylelint reports an issue here - the `label` after the comma just needs to be on the next line. You can confirm locally by running `make lint` in the `es-ds` directory.

Reply:
Cool. Thanks for the detail.
I was in a hurry and didn't even look to see the big red ❌ on CI runs.